### PR TITLE
Input stream

### PR DESCRIPTION
**Pull request by jburel**
Close the input stream

### CODE REVIEW

**Comment by melissalinkert** [comment]
Seems fine, pending builds.  For post-5.1.0, it might make sense to have this class use XMLTools where possible.

**Comment by jburel** [comment]
@melissalinkert: agree that we need to unify that. I did not want to change anything before the "decoupling" work starts.

File: components/specification/src/ome/specification/OmeValidator.java [hunk 1]
```diff
@@ -233,13 +248,20 @@ public Document parseFileWithStreamArrayToSdtErr(File file, StreamSource[] schem
         SchemaResolver theTestClassResolver = new SchemaResolver();
         sFactory.setResourceResolver(theTestClassResolver);
 
-        Schema theSchema = sFactory.newSchema( schemaStreamArray );
-
-        // Version - one step parse and validate (print error to stdErr)
-        dbf.setSchema(theSchema);
+        Schema theSchema = sFactory.newSchema(schemaStreamArray);
+        // Version - two step parse then validate (throws error as exception)
```
Comment (mtbc) [inline components/specification/src/ome/specification/OmeValidator.java]: I assume this change in error handling is okay; does it need mentioning in release notes?

**Comment by mtbc** [comment]
BF builds look shiny and green, code looks good, ready to merge.

File: components/specification/src/ome/specification/OmeValidator.java (hunk 1 above)
Comment (jburel) [inline components/specification/src/ome/specification/OmeValidator.java]: This is mainly/solely used by the tests. (Downgrade tests)

Comment (mtbc) [inline components/specification/src/ome/specification/OmeValidator.java]: :+1:

**Comment by melissalinkert** [comment]
Changes to OmeValidator won't impact 5.1-merge-full-repository, so merging.

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**